pkg/config/users_schema.go

```diff
@@ -0,0 +1,10 @@
+package config
+
+type UserSchema struct {
```

Comment:
feels that all the schemas should be under their own sub directory inside `config`.  So we have a clear separation of the domain